### DISCUSSION TITLE
how to connect different displays with different drivers?

### DISCUSSION THREAD

**Discussion by michcfr** [Bodmer/TFT_eSPI#1483] [Q&A - General] [answered]
Hello
I want to connect my ESP32 to three displays, used simultaneously, each with a different driver: ST7789, ILI9486, GC9A01.
How can I manage three configurations (compile with three User_Setup_Select.h, one for each driver) for the three different displays?

Regards,
Michel

**Comment by Bodmer** [comment, marked as answer]
TFT_eSPI does not support that.

The minimum effort solution is to have one processor per display.  One processor being master and telling the others what to display, say via serial messages via I2C, or UART.  Processors are so cheap and do not need to be of the same type, for example one ESP32 connected to ST7789 and RPi Pico RP2040 (or cheap STM32) based boards for the other displays. For non-WiFi boards the RP2040 ones are my favorite choice due to very large RAM and FLASH.
